src/plugins/vis_type_timeseries/public/application/visualizations/lib/active_cursor.js

```diff
@@ -17,9 +17,8 @@
  * under the License.
  */
 
-// TODO: Remove bus when action/triggers are available with LegacyPluginApi or metric is converted to Embeddable
-import $ from 'jquery';
+import { Subject } from 'rxjs';
 
 export const ACTIVE_CURSOR = 'ACTIVE_CURSOR';
 
-export const eventBus = $({});
+export const eventBus = new Subject();
```

Comment:
- good time to migrate js -> ts
- not sure that `eventBus` is a good name for `rxjs`, especially you use it only for one type of events 
   nit: `activeCursor$ = new Subject<ActiveCursor>();`

Reply:
done